person.rb

```diff
@@ -25,4 +26,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(book, date)
+    Rental.new(date, self, book)
```

Comment:
- [ ] We need to ensure there is uniformity between the parameters defined in the Rental `initialize` method and this one. Kindly do that. Line 31 here should be like so:

  ```
  Rental.new(date, book, self)
  ```